### PR TITLE
Development css

### PR DESCRIPTION
**Pull request by AbuTalha3**
in this capstone project PR I:-
- I selected a Project for an Adventure Sports Competition.
- Made two Pages i.e. Home and About Page. 
- In this PR:-
     - I have done according to the suggestions altogether.
     - The linter is continually giving errors due to the use of scss.
     - Even (npx stylelint "**/*.{css/scss}" ) could not work along with --fix

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,240 +50,199 @@ After you're finished please remove all the comments and instructions!
...
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
+<!-- > Give credit to everyone who inspired your codebase. -->
 
-I would like to thank...
+I would like to thank Microverse for the opportunity to learn.
```
Comment (joskalenda) [inline README.md]: - [x] As required in the project description, please give the acknowledgments to [Cindy Shin in Behance](https://www.behance.net/adagio07) the owner of the design

File: README.md [hunk 2]
```diff
@@ -52,240 +50,199 @@ After you're finished please remove all the comments and instructions!
   - [Install](#install)
   - [Usage](#usage)
   - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
+  - [Deployment](#deployment)
```
Comment (joskalenda) [inline README.md:53]: - [x] Yes please make sure you deploy your application @AbuTalha3, as for now, your readme does not have a demo link provided as required in the project guidelines

File: index.html [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+        <li><a href="#" class="fc-white fs-small no-style">MyPage</a></li>
+      </ul>
+    </div>
+    <div class="nav-items d-flex align-center">
+      <!-- <button type="button" class="menubars" id="Menu-button"> -->
+        <img class="menubars" src="./images/hamburger.png" alt="Menu">
+        <!-- <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span> -->
+      <!-- </button> -->
```
Comment (joskalenda) [inline index.html]: - [x] Please do not leave the commented line of codes in your source code as it makes your code unclean and not professional

File: index.html [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+      <div class="nav-brand">
+        <a href="#"><img src="./images/Summer-Camp.jpg" class="hideOnMobile-f logo" alt="The Summer Camp Logo"></img></a>
+      </div>
+      <div class="nav-links d-right" id="nav-options">
+        <ul class="nav-ul Mobile-Menu gap-wide" id="mobile-Menu">
+          <!-- <li class='d-right close-menu'><button type="button" class="hideOnDesktop-I no-style Dark-Blue" id="closemenu">&times;</button></li> -->
```
Comment (joskalenda) [inline index.html]: - [x] Same as here, please make sure you remove all the commented lines of code

File: index.html [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+          <i class="fas fa-times close crossicon"></i>
+          <li><a class="Dark-Blue fs-medium  no-style" href="About.html">About</a></li>
+          <li><a class="Dark-Blue fs-medium  no-style" href="#Program">Competition</a></li>
```
Comment (joskalenda) [inline index.html]: - [x] If I believe my eyes, there is a type here that makes your about page not to be accessible, you should write `about.html` but instead you did it with a capital `A`

File: style.css
Comment (joskalenda) [inline style.css:1]: - [ ] About the design, I think you need a bit more way to go in changing your colors and fonts, you have done a better presentation but you did not respect the colors, it is required that your design be approximately identical to the original template provided by microverse, please update your page @AbuTalha3 



- [ ] YOUR DESIGN


- [ ] 

- [ ] TEMPLATE DESIGN ASKED TO DO
- [ ] 


- [ ] In your case, you mainly need to make use of these 3 colors only and I think your design will get a better match with the original template

File: index.html [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+  </nav>
+
+  <section class="Headline bg-light">
+    <p class="Headline-greetings">Hello! Adventure Lovers</p>   
+    <h1 class="h1-mobile Headline-title">Summer Camp and Mountain Climbing</h1>
+    <article>
+      A rock climbing competition is to be held 
+      on 26th july 2023 open for all ages. You can register your team with us. We have great prizes 
+      and Surprises both for the winners and participients. 
+    </article>
```
Comment (joskalenda) [inline index.html]: - [x] This is not a proper way of using the article tag, please read this [here](https://www.w3schools.com/tags/tag_article.asp) to have a clear and better understanding of its use, in the meanwhile, I would suggest you change it into a `p` tag

File: index.html [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+          <p class="card-content">Rock Climbing Instructors and C.E.O of the institution. He is a professional
+            Mountaineer and a good singer.
+          </p>
+        </div>
+      </div> 
+      <div class="card">
+        <div class="card-header">
+          <img src="./images/speaker_02.png" class="card-img" alt="Mrs. Anderson">
+        </div>
+        <div class="card-title">
```
Comment (joskalenda) [inline index.html]: - [x] Please make sure you generate this page dynamically using `js` as per the requirement, remove all these hard-coded lines from the HTML, and create an array of objects with speakers data that will be rendered on page load using javascript

File: about.html [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+          <li><a class="Dark-Blue fs-medium  no-style" href="index.html">Home</a></li>
+          <li><a class="Dark-Blue fs-medium  no-style" href="index.html#Fixtures">Fixtures</a></li>
+          <li><a class="Dark-Blue fs-medium  no-style" href="index.html#Guest-Container">Partners</a></li>
+          </ul>
+      </div>
+    </div>
+  </nav>
+
+  <section class="About-Headline">
+    <!-- <p class="Headline-quote">“A younger me needed to climb every mountain. The older me says they photograph better from the bottom. Anonymous”</p> -->
```
Comment (joskalenda) [inline about.html]: - [x] Same comment applies here

File: about.html [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+    <p class="Headline-greeting">Hello! Adventure Lovers</p>
+    <h1 class="h1-mobile About-Headline-title">Summer Camp and Mountain Climbing</h1>
+    <article class="article-about">
+      Welcome to the <strong>The Summer Camp</strong>, where we greet you from closer to nature. 
+      Arranging learning cum adventure for you and your peers that includes close distance kickoffs, 
+      rockclimbing, abceiling, hiking, archery, trout fishing, kyaking, boating, sight seeing, Para Gliding, 
+      and much more. Come and enjoy the cool summers of GB with us.      
+    </article>
```
Comment (joskalenda) [inline about.html]: - [x] The same comment applies here, please change this article tag and use the appropriate `p` tag for this text and remember to read the link I in the above comment

File: about.html [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+          <li><a class="Dark-Blue fs-medium  no-style" href="index.html#Guest-Container">Partners</a></li>
+          </ul>
+      </div>
+    </div>
+  </nav>
+
+  <section class="About-Headline">
+    <!-- <p class="Headline-quote">“A younger me needed to climb every mountain. The older me says they photograph better from the bottom. Anonymous”</p> -->
+    <p class="Headline-greeting">Hello! Adventure Lovers</p>
+    <h1 class="h1-mobile About-Headline-title">Summer Camp and Mountain Climbing</h1>
```
Comment (joskalenda) [inline about.html]: - [x] On this about.html page, we already have an h1 on this line, as per HTML best practice, we are asked to have only one h1 tag for an application, in your case, you have more than 2 h1 tags, please remove and change them to h2 or h3  all the other h1 tags apart from this on line 41

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @AbuTalha3 , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ].. ✅ Green linters
- [ ].. ✅ Great readme file

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — COMMENTED** [review]
#  *Status: Invalid Request* :red_circle: 

Hi @AbuTalha3, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

You have submitted a project, that has linter error. Kindly google search how to resolve this error `Unexpected unknown at-rule "@mixin"   at-rule-no-unknown` online and fix the error. You can [check this link](https://github.com/stylelint/stylelint/issues/3190) and [this](https://stackoverflow.com/questions/60837392/how-can-i-fix-the-css-stylelint-errors-caused-by-sass-scss)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: .stylelintrc.json [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+    ],
+    "scss/at-rule-no-unknown":  [
+      true,
+      {
+        "ignoreAtRules": ["tailwind", "apply", "variants", "responsive", "screen"]
+      }
+    ],
+    "csstree/validator": true,
+    "at-rule-no-unknown": null,
+    "scss/at-rule-no-unknown": true
```
Comment (amejid) [inline .stylelintrc.json]: - [ ] Please do not modify linter configuration files in any way.

**Review by amejid — COMMENTED** [review]
Hi @AbuTalha3, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project, that has a modified linter configuration file. Please use the original configuration file provided by Microverse and solve the linter issues.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: .stylelintrc.json (hunk 11 above)
Comment (AbuTalha3) [reply .stylelintrc.json]: I have tried every way to solve it. Not working 
please guide

File: README.md [hunk 12]
```diff
@@ -52,240 +50,200 @@ After you're finished please remove all the comments and instructions!
...
```
Comment (joskalenda) [inline README.md:53]: - [x] Well done dear @AbuTalha3, you did almost everything, however, as I mentioned earlier, please make sure you deploy your application and add the live demo link to your readme file as stated in the guidelines of the requirements

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @AbuTalha3 , |
|Great work on making the changes requested by a previous reviewer 👏, You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. |
|  |

## Highlights

- [ ].. ✅ Green linters
- [ ].. ✅ Great readme file
- [ ].. ✅ Good design

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @AbuTalha3 , 



In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**